From fcfa6ac4ca21c9331f715ed2dc5f8feebef3754e Mon Sep 17 00:00:00 2001
From: Michael Benfield <mbenfield@google.com>
Date: Sat, 2 Jul 2022 20:18:01 +0000
Subject: [PATCH] Revert "[DebugInfo] Re-enable instruction referencing for
 x86_64"

This reverts commit a03ffad5d34c6d1df14e25df17f18ea2fa437c0c.

The above refers to a commit in the rust-lang clone of the llvm-project
repository.

See b/237879898.


---
 src/llvm-project/llvm/lib/CodeGen/LiveDebugValues/LiveDebugValues.cpp | 5 -----
 2 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/src/llvm-project/llvm/lib/CodeGen/LiveDebugValues/LiveDebugValues.cpp b/src/llvm-project/llvm/lib/CodeGen/LiveDebugValues/LiveDebugValues.cpp
index 40770b15aa35..8f697611a82c 100644
--- a/src/llvm-project/llvm/lib/CodeGen/LiveDebugValues/LiveDebugValues.cpp
+++ b/src/llvm-project/llvm/lib/CodeGen/LiveDebugValues/LiveDebugValues.cpp
@@ -123,11 +123,6 @@ bool LiveDebugValues::runOnMachineFunction(MachineFunction &MF) {
 }
 
 bool llvm::debuginfoShouldUseDebugInstrRef(const Triple &T) {
-  // Enable by default on x86_64, disable if explicitly turned off on cmdline.
-  if (T.getArch() == llvm::Triple::x86_64 &&
-      ValueTrackingVariableLocations != cl::boolOrDefault::BOU_FALSE)
-    return true;
-
   // Enable if explicitly requested on command line.
   return ValueTrackingVariableLocations == cl::boolOrDefault::BOU_TRUE;
 }
-- 
2.37.0.rc0.161.g10f37bed90-goog

